Gemfile

```diff
@@ -32,7 +32,7 @@ gem 'oj'
 # use jquery ui to interact with frontend elements
 gem 'jquery-ui-rails'
 # Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
+gem 'bcrypt', '~> 3.1.7'
```

Comment:
I don't really think that someone will bruteforce one of passwords from my db ever, but scrypt looks really better.